This is the new Master now

Novacat opened the pull request:
Right. This PR has effectively become the new master branch because of permission issues.

Tupinambis committed:
Fully ports https://github.com/Skyrat-SS13/Skyrat-tgmc/pull/7 and adds a fair amount of snowflake customization. Please do not use this power for evil or felinids.

Tupinambis committed:
Enter

Tupinambis committed:
fix

InterroLouis committed:
Nonlethal Marines

Nonlethal options for marines. Includes stamina affecting xenos and the removal of excess stamina damage causing oxyloss.

InterroLouis committed:
Flavor Text

More Flavor Text For...Nonxenos.

InterroLouis committed:
Pizza

Pizza cutting.

Jaybirdyy committed:
Update language_holder.dm

Novacat committed:
Merge branch 'master' into nova-basicfixes

Novacat committed:
[PR to #7] Fixes Marine/Xeno Flavor Text

Fixes the marine/xeno flavor text for the testmerge. The only potential problem is that the previous textboxes had very vigorous sanitation, which it probably needed for a reason. I am not entirely sure why.

Novacat committed:
Fixes sanitizing, fixes records input

Novacat committed:
Fixes

Novacat committed:
stripped_multiline is better

TiviPlus committed:
Initial commit

Novacat committed:
Merges #13

Novacat committed:
Merge remote-tracking branch 'InterroLouis/Nonlethal-Marines' into nova-basicfixes

Novacat committed:
Merge remote-tracking branch 'InterroLouis/Flavor-Text' into nova-basicfixes

Novacat committed:
Merge remote-tracking branch 'InterroLouis/Pizza-Time' into nova-basicfixes

Novacat committed:
Merge remote-tracking branch 'LordDoge/englishbeno' into nova-basicfixes

Novacat committed:
Merge remote-tracking branch 'TiviPlus/Huggerbounty' into nova-basicfixes

Novacat committed:
Fixes Fixes

TiviPlus committed:
Requested changes

chaoko99 commented:
![image](https://user-images.githubusercontent.com/4706052/124699647-0d0cac00-dea0-11eb-9593-01899aabaed5.png)
 An improvement, but it appears to prune breaks and replace them with hashmarks.

chaoko99 commented:
My mistake, it prunes them as part of reading it, likely some kind of excessive sanitation.

Commit:
Reworked Map Rotation

Commit:
Merge pull request #1 from MysteryMan21333/MysteryMan21333-Reworked-Map-Rotation

Reworked Map Rotation

Commit:
Reworked Map Rotation

Commit:
Reworked Map Rotation

Commit:
Reworked Map Rotation

Commit:
Power Monitor Cabling Not Correct (Fixed)

I fixed a mapping issues where the power monitors weren't connected to the power grid.

Novacat commented:
yeah the sanitation is stupidly excessive but I figure there is a reason for it, I would rather not fuck with it personally but if someone has ideas for improvement.

Novacat committed:
Merge remote-tracking branch 'TiviPlus/Huggerbounty' into nova-basicfixes

Novacat committed:
Merge remote-tracking branch 'MysteryMan21333/master' into nova-basicfixes

Novacat committed:
what is this even

Novacat committed:
Fixes Vatborn

Novacat committed:
Fixes experimental welder

fuel_gen is never called so the experimental welder never regens fuel like it is supposed to.

Novacat committed:
Fixes desert uniforms

Novacat committed:
Fixes Desert Uniforms

Novacat committed:
Update sanitize_values.dm
